frame/pools/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "pallet-pools"
```

Comment:
as per discussion offline, I recommend calling this "nomination pools", so you can only pool your nominations in it, not a generic "staking pool".